Web/Edubase.Web.UI/Areas/Groups/ViewRulesHandlers/GroupEditorViewModelRulesHandler.cs

```diff
@@ -18,12 +18,20 @@ public static bool LocalAuthorityIsEditable(GroupEditorViewModel viewModel, IPri
 
         public static bool UserCanCloseAndMarkAsCreatedInError(GroupEditorViewModel viewModel, IPrincipal user)
         {
+            var checkRefreshForCreatedInError = false;
+            if ((!viewModel.StatusId.OneOfThese(eLookupGroupStatus.Closed) && viewModel.CloseAndMarkAsCreatedInErrorLabel == null)
```

Comment:
could this be simplified
something like 
var checkRefreshForCreatedInError = (viewModel.CloseAndMarkAsCreatedInErrorLabel != null) == (viewModel.StatusId.OneOfThese(eLookupGroupStatus.Closed)